charts/osm/values.yaml

```diff
@@ -78,7 +78,7 @@ OpenServiceMesh:
     # -- cert-manager issuer kind
     issuerKind: Issuer
     # -- cert-manager issuer group
-    issuerGroup: cert-manager
+    issuerGroup: cert-manager.io
```

Comment:
@irbekrm could you run `make chart-readme` and commit the diff to this PR. You also need to squash your commits into a single commit for this PR to be able to merge.

Reply:
Thank you! I've done that.